apollo-router/src/cache/storage.rs

```diff
@@ -234,12 +235,12 @@ impl RedisCacheStorage {
         println!("insert result {:?}", r);
     }
 
-    #[cfg(test)]
```

Comment:
If we don't need it we can remove it

Reply:
removed in [dacf151](https://github.com/apollographql/router/pull/2024/commits/dacf1514e317d49cd89785e384a021d83eef3ad5)